fast64_internal/oot/oot_model_classes.py

```diff
@@ -389,15 +389,21 @@ def processDLName(self, name):
         except:
             if name == "gEmptyDL":
                 return None
-            return name
+            else:
+                for attr_name in dir(self.materialContext.ootMaterial.opaque):
+                    if re.match("segment[89ABCD]", attr_name):
+                        setattr(self.materialContext.ootMaterial.opaque, attr_name, False)
+                for attr_name in dir(self.materialContext.ootMaterial.transparent):
+                    if re.match("segment[89ABCD]", attr_name):
+                        setattr(self.materialContext.ootMaterial.transparent, attr_name, False)
+                return name
```

Comment:
would like an explanation

Reply:
I have to admit, I didn't look at it 4Head

Reply:
the loop is different tho, should I zip it?